### PR TITLE
Add option to show all values #292

### PR DESCRIPTION
**Pull request by JackBrian10**

### CODE REVIEW

**Comment by daniele-zurico** [comment]
@JackBrian10 you need to rebase your branch with `release/0.5` I doubt you changed 279 files :) 

commands:
```bash
git checkout release/0.5
git pull
git checkout feature/autocomplete-showAll
git rebase release/0.5
git commit -m 'feat: rebase from 0.5'
git push
```

File: src/autocomplete/Autocomplete.tsx [hunk 1]
```diff
@@ -285,11 +290,14 @@ export const Autocomplete = ({
...
           onChange={handleChange}
-          required = {required}
+          required={required}
+          options={options}
+          showAllValues={showAllValues}
           inputProps={{
             onKeyDown: onKeyDown,
             autoComplete: 'off',
             ...inputProps,
+            showallvalues: showAllValues.toString(),
```
Comment (daniele-zurico) [inline src/autocomplete/Autocomplete.tsx:300]: why do we need to convert it to a string?

File: src/formInput/FormInput.tsx [hunk 2]
```diff
@@ -202,31 +215,89 @@ export const FormInput = ({
...
+              value={value}
+              data-testid="input"
+              onChange={handleChange}
+              required={required}
+              className={inputClassName}
+              aria-label={ariaLabel || name}
+              {...inputProps}
+            />
+            {showAllValues && value === '' && (
+              <FontAwesomeIcon
```
Comment (daniele-zurico) [inline src/formInput/FormInput.tsx:240]: unfortunately this one is not correct... you assuming that everyone will use the icon that you decided. What will be more general purpose is to provide the ability to use a placeholder that will change based on the fact it's clicked or not.
Please remove fontAwesome

**Comment by daniele-zurico** [comment]
@JackBrian10 unfortunately you need to rebase again your branch

**Comment by JackBrian10** [comment]
loco que no me sale

**Pull request closed without merging**